src/Core/Domain/Profile/Command/AddProfileCommand.php

```diff
@@ -51,4 +59,21 @@ public function getLocalizedNames()
     {
         return $this->localizedNames;
     }
+
+    /**
+     * @param string $name
+     */
+    private function assertNameIsStringAndRequiredLength($name)
+    {
+        if (null !== $name && !is_string($name) || strlen($name) > ProfileSettings::NAME_MAX_LENGTH) {
```

Comment:
```suggestion
        if (!is_string($name) || strlen($name) > ProfileSettings::NAME_MAX_LENGTH) {
```
No need to check for null if you check for is_string & strlen :)

Reply:
For TranslatableField type, empty strings are sent as null so I think we need this test

Reply:
Not needed `is_string(null) => false`

Reply:
Yes, that's the problem, we should allow the value null

Reply:
Ooooooooooohhhhhhhhhhhhhhhhhhh :sweat_smile: